### DISCUSSION TITLE
Adding wdio-electron-service fails without disabling sandbox or enabling nodeIntegration

### DISCUSSION THREAD

**Discussion by RedX2501** [webdriverio-community/wdio-electron-service#758] [General]
I'm not that well versed in the whole packaging eco-system of node and electron, so this problem might not be a wdio-electron-service. That's the reason why this is a discussion and not an Issue directly.

I've taken electron-react-boilerplate and added the wdio-electron-service imports into main.ts and preload.ts as documented. I have not wrapped them in the if's because that's not really necessary for this example.

Now when I run the app I receive the following error:

```
VM4 sandbox_bundle:2 Unable to load preload script: C:\data\_eev\electron-react-boilerplate\.erb\dll\preload.js
(anonymous) @ VM4 sandbox_bundle:2
VM4 sandbox_bundle:2 Error: module not found: ./node_modules_wdio-electron-service_dist_preload_js.preload.js
    at preloadRequire (VM4 sandbox_bundle:2:82751)
    at __webpack_require__.f.require (<anonymous>:148:28)
    at <anonymous>:87:40
    at Array.reduce (<anonymous>)
    at __webpack_require__.e (<anonymous>:86:67)
    at <anonymous>:171:21
    at <anonymous>:192:12
    at webpackUniversalModuleDefinition (<anonymous>:8:11)
    at <anonymous>:11:3
    at runPreloadScript (VM4 sandbox_bundle:2:83503)
```

If I enable `nodeIntegration` or disable `sandbox` in the `BrowserWindow` it loads fine.

Is this problem correctly addressed here?

wdio 9 and wdio-electron-service 7.0.1

**Comment by goosewobbler** [comment]
Needing to disable sandbox is a [known workaround](https://github.com/webdriverio-community/wdio-electron-service/blob/main/docs/electron-apis/accessing-apis.md#additional-steps-for-non-bundled-preload-scripts) for being unable to load the preload script, however the documentation could be clearer here; it seems ERB is unable to find the preload when sandbox is enabled despite the preload being bundled with Webpack.  Rather than trawling through Webpack docs to work out why the bundling in ERB is not behaving as I expected, I'll update the service docs to make it more obvious what the fix is, in this case.

**Reply by goosewobbler** [reply]
I added the sandbox disable workaround to common issues: https://github.com/webdriverio-community/wdio-electron-service/blob/main/docs/common-issues-debugging.md#module-not-found-wdio-electron-servicepreload
